### ISSUE TITLE
Thanks

### ISSUE THREAD

**Issue by stuaxo** [TomHarte/CLK#804]
Thanks for this:

"Clock Signal ('CLK') is an emulator for tourists that seeks to be invisible. Users directly launch classic software with no emulator or per-emulated-machine learning curve."

This is *exactly* how I've always thought emulators should work.

Will there eventually be some effort to make windows in GUIS seamless too ?

**Comment by TomHarte** [comment]
Yeah, I've sort of stumbled into the world of windowed apps recently by adding support for the Macintosh and the ST, and it would probably make sense to have a more thorough think about how I really want those to work. The dream would be that application windows aren't tethered to the machines underneath, and that emulated applications can be launched without navigating the classic Finder or equivalent. There might be some sandbox complications if I try to be clever with drives but it'd be worth looking into.

That said, I'm also in a weird position because the emulator attempts to do a cycle-accurate emulation of the Macintosh, which isn't really all that helpful to a user. So another direction of travel I'd like to take is to explore inaccurate-but-fast emulation of later Macintoshes, at least up to the 68040s, in which case it might make sense to look more closely at not boxing emulated applications in alongside that — if the code is structured more as constructing a version of the classic Macintosh environment rather than necessarily a specific machine.

So, thoughts still quite nebulous, and I've currently no idea how I would achieve these things, but I'd be a big fan of them as a user.

**Comment by Tiza59** [comment]
Thank you very much for this incredible piece of software. Please give us a Paypal-Link or something, where we can send you a little gift in recognition for your excellent work.
